devops/src/superai/paidy_super_ai.py

```diff
@@ -0,0 +1,14 @@
+from robot import Robot
+
+
+def calculate_risk_score(email_address: str) -> float:
+    """Calculates a score between 0.0 (high risk) and 1.0 (low risk) for a given consumer email-address.
+
+    Raises an exception in case provided email-address is invalid.
+    """
+    pass
```

Comment:
I'm confused are we expecting them to add documentation? Why are the parameters not documented here??

Reply:
It just doesn't matter, and I don't want to distract from the actual project / goal. And I don't want to waste time on this, at least one candidate is probably already waiting for the assignment.

For the parameters: there are different ways/formats for how to document arguments. I tested some and pydoc seems not to care at all. It just displays the doc-string. As it does not really matter for the assignment, I'd like to reduce confusion on this and leave it out.

Reply:
I see your point.